x-pack/osquerybeat/Jenkinsfile.yml

```diff
@@ -19,8 +19,11 @@ stages:
           make -C x-pack/osquerybeat update;
           make check-no-changes;
         stage: lint
-    build:
-        mage: "mage build test"
+    unitTest:
+        mage: "mage build unitTest"
+        stage: mandatory
+    goIntegTest:
+        mage: "mage goIntegTest"
```

Comment:
unrelated' I see a couple of commits around testing that don't belong here I think